src/sentry/api/endpoints/event_owners.py

```diff
@@ -21,25 +20,20 @@ def get(self, request, project, event_id):
         :pparam string event_id: the id of the event.
         :auth: required
         """
-
-        use_snuba = options.get('snuba.events-queries.enabled')
-
-        event_cls = SnubaEvent if use_snuba else Event
-
-        event = event_cls.objects.from_event_id(event_id, project.id)
+        event = SnubaEvent.objects.from_event_id(event_id, project.id)
         if event is None:
             return Response({'detail': 'Event not found'}, status=404)
 
-        # populate event data
-        Event.objects.bind_nodes([event], 'data')
-
         owners, rules = ProjectOwnership.get_owners(project.id, event.data)
 
         # For sake of the API, we don't differentiate between
         # the implicit "everyone" and no owners
         if owners == ProjectOwnership.Everyone:
             owners = []
 
+        # populate event data
```

Comment:
this was happening before 
> ProjectOwnership.get_owners(project.id, event.data)
which depends on event.data so potentially depended on data filled in by bind_nodes.

Reply:
Ah thanks, put this back in the wrong place